pkg/server/sotw/v2/server.go

```diff
@@ -162,7 +120,7 @@ func (s *server) process(stream Stream, reqCh <-chan *discovery.DiscoveryRequest
 	}()
 
 	// sends a response by serializing to protobuf Any
-	send := func(resp cache.Response, typeURL string) (string, error) {
+	send := func(resp cache.Response) (string, error) {
 		if resp == nil {
 			return "", errors.New("missing response")
```

Comment:
This is an incorrect error message. `nil` is an intentional error not a missing response.